### PR TITLE
Get alerts button experiment

### PR DESCRIPTION
**Pull request by chasulin**
[Figma mocks of mobile-only experiment variation](https://www.figma.com/file/JGqemgZwlG9e5DQbZpdKgy/Redesign?node-id=1575%3A104549)

### CODE REVIEW

**Comment by chasulin** [comment]
[Implement alert signup design as experiment on existing location page [mobile only]](https://trello.com/c/0OdNGiOB/1218-implement-alert-signup-design-as-experiment-on-existing-location-page-mobile-only)

File: src/components/LocationPage/Experiment/GetAlertsButton.style.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+import fonts from 'common/theme/fonts';
```
Comment (chasulin) [inline src/components/LocationPage/Experiment/GetAlertsButton.style.tsx:2]: ill move this into the theme properly in a diff pr

File: src/components/LocationPage/Experiment/GetAlertsButton.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import {
+  Button,
+  Arrow,
+  EmailIconWrapper,
+  Row,
+  Column,
+  Subtext,
+} from './GetAlertsButton.style';
+
+const GetAlertsButton: React.FC<{ onClick: any }> = ({ onClick }) => {
```
Comment (pnavarrc) [inline src/components/LocationPage/Experiment/GetAlertsButton.tsx:12]: nit: fix the `any`

File: src/components/LocationPage/Experiment/LocationPageHeaderVariant.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    stats,
+    onMetricClick,
+    onHeaderShareClick,
+    onHeaderSignupClick,
+    isMobile,
+    region,
+  };
+
+  return (
+    <Experiment id={ExperimentID.MEGA_MENU_API_LINK}>
```
Comment (pnavarrc) [inline src/components/LocationPage/Experiment/LocationPageHeaderVariant.tsx:42]: Is this the wrong id?

File: src/components/LocationPage/Experiment/LocationPageHeaderVariant.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      <Variant id={VariantID.B}>
+        <LocationPageHeader {...headerProps} />
+      </Variant>
+      <Variant id={VariantID.A}>
+        <LocationPageHeader {...headerProps} showNewButton={true} />
+      </Variant>
```
Comment (pnavarrc) [inline src/components/LocationPage/Experiment/LocationPageHeaderVariant.tsx:48]: Also, is this reversed? Shouldn't be A the original version?

Comment (chasulin) [reply src/components/LocationPage/Experiment/LocationPageHeaderVariant.tsx:48]: yeah i just set the new version as A for the sake of QA. will switch back before merging

**Review by pnavarrc — APPROVED** [review]
Looks good! The new email alerts button has ripple effect, maybe we should remove it before shipping

**Pull request merged**